plover/gui_qt/main_window.py

```diff
@@ -238,6 +243,15 @@ def on_disable_output(self):
     def on_configure(self):
         self._configure()
 
+    def on_open_config_folder(self):
+        # webbrowser.open does not work on Mac so we need OS-specific logic
```

Comment:
I think you can remove the mention, as testing it again on Linux with my configuration, I see that it always open a directory listing in my browser, which is clearly not the best behaviour.